docs/src/literate-tutorials/gpu_qp_heat_equation.jl

```diff
@@ -0,0 +1,265 @@
+import Pkg;
+Pkg.activate("")
+Pkg.instantiate()
```

Comment:
```suggestion
```

No need for this on every run. Just add CUDA to the docs project and also commit the Manifest.toml.